src/components/core/Input/Input.tsx

```diff
@@ -19,8 +20,19 @@ export interface Props
   containerProps?: StackProps;
 }
 
-const PandaInput = panda(ark.input, input);
-const PandaLabel = panda(ark.label);
+const { withProvider, withContext } = createStyleContext(input);
+
+// TODO extract primitives, add prop interfaces, and export both (this entire component will be refactored)
+
+const InputLabel = withContext(panda(ark.label), "label");
+
+const InputAddon = withContext(panda.div, "addon");
+
+const InputLeftElement = withContext(panda.div, "leftElement");
```

Comment:
The default for `InputLeftElement` in SB is messed up. It renders outside of the input

Reply:
Yes, I made a note of this here: https://github.com/animareflection/ui/pull/197#discussion_r1438109920

Reply:
I can fix in this PR if you want, but it is caused by the styles now correctly being applied. This can be verified otherwise by e.g. checking that now the correct brand color focus border is applied in this PR (vs. e.g. the production Storybook deployment)